packages/badge/src/Badge.ts

```diff
@@ -35,21 +36,70 @@ export const BADGE_VARIANTS = [
     'yellow',
 ] as const;
 export type BadgeVariant = typeof BADGE_VARIANTS[number];
+export const FIXED_VALUES_DEPRECATED = ['top', 'bottom', 'left', 'right'];
+export const FIXED_VALUES = [
+    'fuchsia',
+    'indigo',
+    'magenta',
+    'purple',
+    'seafoam',
+    'yellow',
+] as const;
```

Comment:
🤣 that's definitely a mistake, haha! Code review for the win. I'll get this corrected later today. 🙈